example/lib/splash_screen.dart

```diff
@@ -0,0 +1,81 @@
+import 'package:amity_uikit_beta_service/amity_uikit.dart';
+import 'package:amity_uikit_beta_service_example/main.dart';
+import 'package:flutter/material.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+
+class SplashScreen extends StatefulWidget {
```

Comment:
Implement the splash screen to setup UIKit using cache from shared preference. Can hard code to the setup process if the client doesn't have dynamic apikey and endpoint.